spanner_orm/field.py

```diff
@@ -219,9 +230,20 @@ def validate_type(self, value: Any) -> None:
 class BytesBase64(FieldType):
   """Represents a bytes type that must be base64 encoded."""
 
+  def __init__(self, size: Optional[int] = None):
+    """Initializer.
+
+    Args:
+      size: Size of the Byte64. MAX is used if not specified.
+    """
+    if size is not None and size <= 0:
+      raise error.ValidationError('Byte64 size must be positive')
```

Comment:
(nit) The underlying type is BYTES, not BYTE64. The 64 doesn't really make any sense outside of the context of base64 encoding. Since the length is the number of bytes rather than the length of the base64 encoding, these should probably talk about it as the length of the bytes rather than anything about base64.

Reply:
Thanks for the background!